benchmark/lib/Streamly/Benchmark/Data/Fold.hs

```diff
@@ -208,3 +210,18 @@ o_n_heap_serial_folds value =
                 ]
           ]
     ]
+
+o_1_space_serial_parse :: Int -> [Benchmark]
+o_1_space_serial_parse value =
```

Comment:
Not so important, but you could move this to `benchmark/lib/Streamly/Benchmark/Data/Parse.hs`.

Reply:
Done.